eos/b-decays/observables.cc

```diff
@@ -56,6 +57,32 @@ namespace eos
 
         return ObservableGroup(imp);
     }
+
+    ObservableGroup
+    make_b_to_3l_nu_group()
+    {
+        auto imp = new Implementation<ObservableGroup>(
+            R"(Observables in $B^-\to \ell_\prime^+\ell_\prime^-\ell^-\bar\nu$ decays)",
```

Comment:
Maybe it would be good to follow the order in the paper, i.e., move the primed leptons to the right of the unprimed ones?